server/api/urls.py

```diff
@@ -3,23 +3,35 @@
 from . import views
 
 urlpatterns = [
+
+# Active Endpoints
+########################################################################################
     path('admin/', admin.site.urls), 
 
+    # POST Endpoints
+    path('postCourse2023', views.postCourse2023),
+
+    # GET Endpoints
+
```

Comment:
Are we adding these in later as we refactor them?

Reply:
Yes that was my plan, there were just lots of clutter.